src/components/CapInputLabel.vue

```diff
@@ -0,0 +1,99 @@
+<template>
+  <div :class="['cap-ui', 'cap-input cap-label-input', { 'has-error': error }]">
+    <label
+      v-if="label"
+      class="outer-label"
+      :class="{ active: focused || prefiled }"
+    >
+      {{ label }}
+    </label>
+    <div class="input-container-wrapper">
+      <div
+        class="cap-input-label-container"
+        :class="[{ disabled: disabled, rounded: rounded }, size]"
+      >
+        <label
+          v-if="label"
+          class="label"
+          :class="{ active: focused || prefiled }"
+        >
+          {{ label }}
+        </label>
+        <input
```

Comment:
Why not re-use `cap-input` component here?

Reply:
I have replaced it with `cap-input`